.github/workflows/helm-deploy-eks.yml

```diff
@@ -58,6 +58,11 @@ on:
         description: 'Environment for deployment'
         required: true
         type: string
+      timeout:
+        description: 'Duration to wait for a successful deployment'
```

Comment:
Super minor but should we clarify if this is seconds or milliseconds?

Reply:
Yeah, that's a great idea. Updated.